src/transformers/models/t5/modeling_t5.py

```diff
@@ -262,6 +262,8 @@ def forward(self, hidden_states):
         return self.weight * hidden_states
 
 
+ALL_LAYERNORM_LAYERS.append(T5LayerNorm)
```

Comment:
This should be after the try except block below, which may change `T5LayerNorm`.